Fix globals, part 6: `performance` (#33339)
mdn/content: files/en-us/web/api/performance_api/performance_data/index.md

## Performance data structure

> The Performance API collects performance data for each global object (`window.performance` or `self.performance` in workers). See {{domxref("window.performance")}} for which APIs are available in window and worker contexts. If you are collecting performance metrics for multiple contexts, also take a look at {{domxref("performance.timeOrigin")}} to synchronize time origins between contexts.

Reviewer: ```suggestion
With the Performance API you can collect performance data in both {{domxref("Window.performance", "Window")}} and {{domxref("WorkerGlobalScope.performance", "Worker")}} global contexts. If you are collecting performance metrics for multiple contexts, take a look at {{domxref("performance.timeOrigin")}} to synchronize time origins between contexts.
```
